charts/beyla/templates/daemon-set.yaml

```diff
@@ -62,6 +60,7 @@ spec:
                 - CHECKPOINT_RESTORE
                 - DAC_READ_SEARCH
                 - PERFMON
+                - NET_ADMIN
```

Comment:
I wonder if we need to create a new option to disable context propagation that will reduce the permissions back, remove host network and the extra volume mounts?